WordPress/wordpress-playground
# Test/hello world blueprint

This is a demo, it shouldn't be merged.

## What is this PR doing?

Creates a `helloWorld` blueprint step and executes PHP code to build a response message.

## What problem is it solving?

Demonstrating how to add blueprint steps.

## How is the problem addressed?

By adding a new blueprint step that executes code in base-php.

## Testing Instructions

- checkout this branch
- run `npm run dev`
- Open the [local site](http://localhost:5400/website-server/)
- See log message in dev tools _PHP: Hello World!_
